.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+
+**What type of PR is this?**
+> Uncomment only one:
```

Comment:
Also mention to add the appropriate GitHub label

Reply:
done